wp-content/themes/csisnuclearnetwork/assets/_scss/components/_team-member-block.scss

```diff
@@ -0,0 +1,82 @@
+@use '../abstracts' as *;
+
+.wp-block-lazyblock-team-members {
+  .team-member-block__ul {
+    padding-left: 0;
+  }
+
+  .team-member-block__ul li {
+    padding-bottom: rem(20);
+    padding-left: 0;
+    @include breakpoint('medium') {
+      display: flex;
+    }
+  }
+
+  .team-member-block__name {
+    @extend %text-style-heading-medium;
+    color: $color-accent-primary-blue-200;
+    font-weight: bold;
+    line-height: 100%;
+  }
+
+  .team-member-block__title {
+    @extend %text-style-label-medium;
+    display: inline-block;
+    color: $color-black-165;
+    line-height: 100%;
+
+    &::after {
+      content: '';
+      display: block;
+      width: rem(45);
+      height: rem(4);
+      margin-top: rem(8);
+      margin-bottom: rem(12);
+      background-color: $color-accent-secondary-yellow-100;
+    }
+  }
+
+  .team-member-block__bio {
+    @extend %text-style-paragraph-medium-short;
+    color: $color-black-190;
+  }
+    
+  .senior-1 {
+    padding-left: 0;
+
+    .team-member-block__name {
+      @extend %text-style-heading-large-strong;
+      line-height: 100%;
+    }
+
+    .team-member-block__title {
+      @extend %text-style-label-large;
+      line-height: 100%;
+    }
+  }
+
+  .team-member-block__img {
+    float: left;
+    width: rem(80);
+    max-width: none;
+    height: rem(100);
+    margin-right: rem(16);
+    margin-bottom: rem(16);
+    object-fit: cover;
+
+    @include breakpoint('medium') {
+      width: rem(100);
+      height: rem(125);
+      margin-right: rem(24);
+      margin-bottom: 0;
+    }
+
+    @include breakpoint('large') {
+      width: rem(145);
+      height: rem(180);
+      margin-bottom: 0;
```

Comment:
You can remove this, since it's included in the medium breakpoint.

Reply:
Do you mean just the `margin-bottom: 0` line?

Reply:
Yes, just the margin-bottom